ubireader/ubifs/list.py

```diff
@@ -39,9 +40,10 @@ def list_files(ubifs, list_path):
 
         if len(inodes) < 2:
             raise Exception('No inodes found')
+        # if encrypted decrypt filenames
+        decrypt_filenames(inodes)
```

Comment:
It would be cleaner if decryption was happening when nodes are parsed rather than a post-processing step.